File: codekata/basics/area_of_a_triangle.py
# area of a triangle
# get base and height as inputs from user in a single line
base,height=list(map(int,input().split()))
# define area triangle function
def area_triangle():
# area of a triangle formula
      area=(1/2*base*height)
# return area
      return area
# call area triangle function and store it in result
result=area_triangle()
# print result value
print(result)

File: codekata/javascript/basics/area_of_a_triangle.js
// Given base(B) and height(H) of a triangle find its area
const readline = require('readline');
const inp = readline.createInterface({
  input: process.stdin
});
const userInput = [];
inp.on("line", (data) => {
userInput.push(data);
});
inp.on("close", () => {
	var number=userInput[0].split(' ');
	var base=number[0],height=number[1];
	var area=(base*height)/2;
	console.log(area);
});
